spring-boot/admin/src/main/java/eu/coatrack/admin/GitHubSecurityConfig.java

```diff
@@ -48,7 +37,6 @@ protected void configure(HttpSecurity http) throws Exception {
                 .antMatchers(PERMITALL_RESOURCE_LIST).permitAll()
                 .anyRequest().authenticated()
                 .and()
-                .addFilterBefore(new PublicApiTokenAccessFilter(tokenServices), AbstractPreAuthenticatedProcessingFilter.class)
```

Comment:
No, it means that this will be solved on the followup issue coatrack#183.
I already created a branch from this branch to then work on it

Reply:
What I mean is: if we merge _this_ version and deploy it, then it won't work, right? So we also need to solve that follow-up issue before everything works as before.

Reply:
So, if we merge this the Oauth2 will work, just not the authentication via bearer Token on the header to the public api

Reply:
Yes, exactly. So I will solve the 183 specifically on another branch and merge to this branch.
Because if I decided not to do a follow up, then this branch would become much bigger and confusing as the issue related to this PR is to "solve the DummyClientRegistrationRepository"